db/blip_handler.go

```diff
@@ -1189,7 +1226,9 @@ func (bh *blipHandler) processRev(rq *blip.Message, stats *processRevStats) (err
 	// If the doc is a tombstone we want to allow conflicts when running SGR2
 	// bh.conflictResolver != nil represents an active SGR2 and BLIPClientTypeSGR2 represents a passive SGR2
 	forceAllowConflictingTombstone := newDoc.Deleted && (bh.conflictResolver != nil || bh.clientType == BLIPClientTypeSGR2)
-	if bh.conflictResolver != nil {
+	if bh.activeCBMobileSubprotocol >= CBMobileReplicationV4 {
+		_, _, _, err = bh.collection.PutExistingCurrentVersion(bh.loggingCtx, newDoc, incomingHLV, rawBucketDoc)
+	} else if bh.conflictResolver != nil {
```

Comment:
Given that PutExistingRev is just a wrapper for PutExistingRevWithConflictResolution (that passes a nil conflict resolver), I think we could simplify this in future by just calling PutExistingRevWithConflictResolution directly here for the non-CV case.  I was going to put a TODO, but I think this will be obvious when someone comes along and adds conflict resolution for CBMobileReplicationV4 in future.